.changeset/silly-hats-end.md

```diff
@@ -0,0 +1,8 @@
+---
+"@fuel-ts/abi-coder": patch
+"@fuel-ts/abi-typegen": patch
+---
+
+feat: add missing support for the `u256` type
+
+The TS SDK is now capable of handling `u256` types, and hence capable of interacting (encoding/decoding) with any Sway programs that use the `u256` type.
```

Comment:
We could also have a "breaking change" section with the changeset so it'll make it to the changelog.